actions/redoc/template.hbs

```diff
@@ -684,18 +718,29 @@
     .crBktj .token.selector, .crBktj .token.attr-name, .crBktj .token.string, .crBktj .token.char, .crBktj .token.builtin, .crBktj .token.inserted{color:#416581}
     .crBktj .token.boolean{color:#1878e0}
     .jnkRQj .collapser{color:#000000}
-
     div.react-tabs__tab-panel > div > div > span{color:#416581}
     div.react-tabs__tab-panel > div > div > div > span{color:#416581}
     ul > li.react-tabs__tab.react-tabs__tab--selected{color:#ecf0f5}
     ul > li.tab-success.react-tabs__tab--selected{color:#F6FFF4}
     ul > li.tab-error.react-tabs__tab--selected{color:#fbf1f0}
     .react-tabs__tab-panel > div > pre > .token.string, .react-tabs__tab-panel > div > pre > .token.builtin{color:#416581}
-
     .react-tabs__tab-panel > div > pre > .token.operator,
     .react-tabs__tab-panel > div > pre > .token.variable{color:#324350}
     div.react-tabs__tab-panel.react-tabs__tab-panel--selected{border-radius:10px}
   </style>
 </body>
 
-</html>
\ No newline at end of file
+</html>
+© 2021 GitHub, Inc.
+Terms
+Privacy
+Security
+Status
+Docs
+Contact GitHub
+Pricing
+API
+Training
+Blog
+About
+Loading complete
```

Comment:
???

Reply:
yea I feel like this is some weird auto-addition which I hadn't seen, since I didn't add this manually